.vscode/scripts/run-mosquitto.sh

```diff
@@ -14,18 +14,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 echo "#######################################################"
-echo "### Running Mosquitto                               ###"
+echo "### Running MQTT Broker                             ###"
 echo "#######################################################"
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-#Terminate existing running VAL services
-RUNNING_CONTAINER=$(docker ps | grep "eclipse-mosquitto" | awk '{ print $1 }')
-MOSQUITTO_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.version | tr -d '"')
+MOSQUITTO_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.image | tr -d '"')
+MOSQUITTO_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.version | tr -d '"')
```

Comment:
same, here, should we use app-manifest?

Reply:
See comments to prerequisite_settings.json